plugin/pkg/admission/resourcequota/admission.go

```diff
@@ -77,6 +77,9 @@ var resourceToResourceName = map[unversioned.GroupResource]api.ResourceName{
 	api.Resource("resourcequotas"):         api.ResourceQuotas,
 	api.Resource("secrets"):                api.ResourceSecrets,
 	api.Resource("persistentvolumeclaims"): api.ResourcePersistentVolumeClaims,
+	extensions.Resource("replicasets"):     extensions.ResourceReplicaSets,
```

Comment:
@derekwaynecarr Are there any non-admin namespaced resources we don't want in this list?

What about ConfigMap? HPA?